### PR TITLE
SAK-43324 - T&Q - Inconsistent display of tab options in Scores screens

### PR DESCRIPTION
**Pull request by mpellicer**
https://jira.sakaiproject.org/browse/SAK-43324

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/evaluation/detailedStatistics.jsp [hunk 1]
```diff
@@ -35,6 +35,12 @@ $Id: histogramScores.jsp 38982 2007-12-06 13:05:38Z gopal.ramasammycook@gmail.co
 **********************************************************************************/
 --%>
 -->
+<script>
+  $(document).ready(function(){
+    $('#histogram\\:detailedStatisticsMenuLink').addClass('current');
```
Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/jsf/evaluation/detailedStatistics.jsp]: can you add this class via JSF attribute?

Comment (mpellicer) [reply samigo/samigo-app/src/webapp/jsf/evaluation/detailedStatistics.jsp]: You got me!

I tried many ways, via includes and via beans, didn't find a way to do it without using facelets :( That's why I used Javascript. Do you know a way to do it via pure mojarra JSF?

Comment (ottenhoff) [reply samigo/samigo-app/src/webapp/jsf/evaluation/detailedStatistics.jsp]: it's not just a simple styleClass?

Comment (mpellicer) [reply samigo/samigo-app/src/webapp/jsf/evaluation/detailedStatistics.jsp]: The problem is that the JSF template should tell the JSF menu template which tab is the current one, and the "include" directive we're using doesn't allow parameters. That's why I added a JS snippet to apply the current class to the tab, the template tells the menu where the user is.

If you know an alternative approach I'll be glad to apply it!

Thanks for the review @ottenhoff

**Pull request merged**